### DISCUSSION TITLE
Readme for organization front page?

### DISCUSSION THREAD

**Discussion by simonc8** [community/community#23590] [New to GitHub]
<p>Is there a way to make a readme which will be visible when someone goes to an organization front page?</p>
<p>This would say a bit about the organization and give a brief description of the projects. I know you can make a profile for an individual but I can’t see how to do something similar for an organization.</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey! This is not currently possible. The only thing you can do would be set up the organization’s profile which can provide limited info for those visiting the org. You can include a basic description, logo, website, location, and email. You can also pin a specific repo to the org’s repo section. </p>

**Comment by simonc8** [comment]
<p>Thanks for confirming that. It would be a nice feature to have.</p>

**Comment by JustinGOSSES** [comment]
<p>Just wanted to 2nd that this would be a nice feature to have.</p>
<p>I created one thinking it would work as there is nothing in the instructions that explicitly say it won’t work for Orgs. <a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-profile/managing-your-profile-readme" class="inline-onebox" rel="noopener nofollow ugc">Managing your profile README - GitHub Docs</a></p>

**Comment by drguthals** [comment]
<p>+1 would love this support :slight_smile:</p>

**Comment by gvegayon** [comment]
<p>Actually, I think I found a bot, as, by the time I created one for my organization, I got the message saying, “you have found a secret.”</p>

**Comment by jlengrand** [comment]
<p>Hey there,</p>
<p>What is the status of this issue? <a class="mention" href="/gvegayon">@gvegayon</a> seems to mention that it worked for him?<br>
I have just tried, but with no luck so far.</p>
<p>And as <a class="mention" href="/justingosses">@JustinGOSSES</a> mentions, the documentation does not say anything about Organization not being supported at the moment, so that’d be helpful to add indeed in case it is not</p>

**Comment by gvegayon** [comment]
<p>Ups, I meant BUG not BOT, so it is not working for me :stuck_out_tongue:</p>

**Comment by jlengrand** [comment]
<p>hahaha, alright, that makes more sense thanks!</p>

**Comment by eeholmes** [comment]
<p>This is now possible at the organization level.</p>
<ol>
<li>Create a repo called .github</li>
<li>In that create a folder call profile</li>
<li>Within that put your Readme.md</li>
</ol>
<p>Actually as soon as you make the .github repo, GitHub will alert you that this is special and will walk you through the next steps.</p>

**Reply by Aloagbaye** [reply]
This  worked for me. Thank you @eeholmes

**Comment by silkfire** [comment]
<p>Is there a reason why this feature doesn’t exist in GitHub Enterprise yet?</p>

**Reply by pserwylo** [reply]
This is available in GHE as of v3.4. See release notes here: https://docs.github.com/enterprise-server@3.4/admin/release-notes#organization-changes.

I didn't realise that it had to live in a `profile/` subdirectory of the `ORG_NAME/.github` repository. I noticed that when reading the linked docs from the changelog: https://github.blog/changelog/2021-09-14-readmes-for-organization-profiles/.
